lib/slack.py

```diff
@@ -20,13 +19,27 @@ def get_client(self):
         if self._client is not None:
             return self._client
 
-        token = Config.get_variable('tubey_credentials', 'bot_oauth_token')
+        token = Config.get_variable('tubey', 'bot_oauth_token')
         sc = SlackClient(token)
         self._client = sc
         return self._client
 
+    def send_video(self, video_id):
+        # Sends a video in the current channel using the video id
+        video_url = "https://www.youtube.com/watch?v={}".format(video_id)
+        self.send_message(video_url)
```

Comment:
You need to change this to `{'text': video_url}` since we got rid of the `message` parameter